sphinx_gallery/gen_gallery.py

```diff
@@ -20,6 +20,12 @@
 from .docs_resolv import embed_code_links
 from .downloads import generate_zipfiles
 
+try:
+    FileNotFoundError
+except NameError:
+    # Python2
+    FileNotFoundError = IOError
```

Comment:
I forgot to tell you when I saw that in a different PR but this is a slightly WTF. Just raise IOError this way you don't have to do a Python2/3 dance.